modules/ROOT/pages/mutations/delete.adoc

```diff
@@ -74,8 +74,7 @@ mutation {
 }
 ----
 
-By the time the traversal has reached it, that empty `where` argument has the context of only refer to posts that were created by Jane Doe, as the traversals to those `Post` nodes were from her `User` node. 
-Essentially, the above query is equivalent to:
+This query is equivalent to:
```

Comment:
TBF, I believe this last example add more confusion that benefit, not a as part of you're PR but I would delete this part entirely, I'll ping @mjfwebb and @angrykoala for a couple of extra eyes here.

Reply:
Yes I agree, after a small discussion we can just get rid of this last part.

Reply:
probably relates to what @mjfwebb posted in the docs-graphql channel: https://neo4j.slack.com/archives/C03E32T5MCL/p1729502123351959

so, get rid of the whole example then?

Reply:
In my opinion yes, I would get rid of the `This query is equivalent to:`  with the following example

Reply:
done